crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -161,12 +164,21 @@ public Response insert(InsertionRequest req) {
                 ctx.getHookManager().callQueuedHooks();
             }
         } catch (Error e) {
+            if(metricCtx!=null) {
+                metricCtx.markRequestException(e);
+            }
             response.getErrors().add(e);
             response.setStatus(OperationStatus.ERROR);
         } catch (Exception e) {
+            if(metricCtx!=null) {
+                metricCtx.markRequestException(e);
```

Comment:
I too feel this should give more context information in each metric and will handle errors metrics more efficiently. Modified this PR accordingly.